cli/find.php

```diff
@@ -146,5 +144,20 @@
 }
 
 $progress->update_full(100, "Finished searching into $output");
+
 fclose($fp);
+
+if ($summary) {
+    // Read the content of the output file.
```

Comment:
this all looks weird to me, whats the reason for this? we really don't want to touch the file a second time and we definitely don't want to load it all into memory at once